plugin/core/registry.py

```diff
@@ -105,8 +104,20 @@ def sessions(self, capability: Optional[str] = None) -> Generator[Session, None,
         yield from sessions_for_view(self.view, capability)
 
 
-class LspRestartClientCommand(sublime_plugin.TextCommand):
+class LspRestartServerCommand(LspTextCommand, sublime_plugin.TextCommand):
```

Comment:
LspTextCommand already inherits the sublime text command so drop the second argument